### PR TITLE
Gaming Workload Scenarios #1 - Minecraft

### LINKED ISSUE #445: Workload gaming scenarios
Add in a new scenario to the dropdown for Gaming Workloads.
This will provide some "fun" into the AKSC experience.

It will require the configpreset.json file schema to be extended to cope with additional post deploy commands (misc infra and workload installation into aks)

### PR DESCRIPTION
**Pull request by Gordonby**
### Gaming preset

Added a new preset scenario for gaming. 2 cards have been added for Minecraft.

> Note that the script we'll need for Minecraft workload will be considerably shorter when bicep extensibility for kubernetes comes out. (It'll drop down to 1 line!)

Closes #445 

image

image

#### TODO

- [x] Add a preset description h2 to the page
- [x] Implement card level post deploy additions (for workload specific scripts) using token replacement for ResourceGroup name
- [x] Support public ip per node #452 

#### vm sku chmod
WSL/Windows is messing up the attributes of the `generate-vm-sku-list-v2.sh` file. For reliability, we need to chmod it during the workflow.

### eslint

Incorporated eslint fixes for reported warnings

## PR Checklist

- [x] PR has a meaningful title
- [x] Summarized changes
- [x] This PR is ready to merge and is not **Work in Progress**
- [x] Link to a filed issue
- [x] Screenshot of UI changes (if PR includes UI changes)

### CODE REVIEW

**Comment by naioja** [comment]
@Gordonby specifically for gaming workloads we would need to enable IPs per node for lower latency and direct client connects.

Reference documentation: 
https://learn.microsoft.com/en-us/azure/aks/use-multiple-node-pools#assign-a-public-ip-per-node-for-your-node-pools
https://github.com/PlayFab/thundernetes/blob/main/docs/quickstart/installing-aks.md
https://agones.dev/site/docs/installation/creating-cluster/aks/#creating-the-aks-cluster

**Comment by Gordonby** [comment]
> @Gordonby specifically for gaming workloads we would need to enable Public IPs per node for lower latency and direct client connectivity.
> 
> Reference documentation: https://learn.microsoft.com/en-us/azure/aks/use-multiple-node-pools#assign-a-public-ip-per-node-for-your-node-pools https://github.com/PlayFab/thundernetes/blob/main/docs/quickstart/installing-aks.md https://agones.dev/site/docs/installation/creating-cluster/aks/#creating-the-aks-cluster

Captured this in #452

**Comment by Gordonby** [comment]
`Feedback`

title font too big
workload deploy shellscript box

File: helper/src/configpresets/gaming.json [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    "gaming": {
+        "title": "Gaming",
+        "description": "Gaming Scenarios for Kubernetes that include end to end workload installation",
+        "icon": "Game",
+        "sections": [
+            {
+                "key": "minecraft",
+                "sectionTitle": "Minecraft servers",
+                "sectionDescription" : "Minecraft servers allow you to create your own internet accessible world to play with your friends online.",
+                "sectionWarning": "Whilst Minecraft servers are free, players using Minecraft requires a licence",
```
Comment (MattLeach25) [inline helper/src/configpresets/gaming.json:11]: require*

Comment (Gordonby) [reply helper/src/configpresets/gaming.json]: done!

**Review by MattLeach25 — APPROVED** [review]

**Pull request merged**